docs/releases/1.9.2.txt

```diff
@@ -71,3 +71,7 @@ Bugfixes
 
 * Fixed a crash when using a reverse ``OneToOneField`` in
   ``ModelAdmin.readonly_fields`` (:ticket:`26060`).
+
+* Fixed a crash when calling the ``migrate`` command in a test case with the
```

Comment:
The only failure I could reproduce was with [`swappable_models.tests.SwappableModelTests.test_generated_data`](https://github.com/django/django/blob/275d11fbc5e8f664552a458ce141d4394fa70e1f/tests/swappable_models/tests.py#L12-L30) when I tried running the test suite with simply replacing the bogus migration modules entry in `runtests.py` by `None`.

What's odd is that even if this attribute is only documented to work with `TransactionTestCase` it's used on a `TestCase` here and the failure disappear (even without the fix to the questioner) if the attribute is removed. Hence my conclusion here that the issue must be related to interactions between the three features.